testing-rspec/README.md

```diff
@@ -31,6 +31,8 @@
 - Use a [Fake] to stub requests to external services.
 - Use integration tests to execute the entire app.
 - Use non-[SUT] methods in expectations when possible.
+- Use shared examples to test behavior that should be the same across different
+  implementations (React/Rails).
```

Comment:
Other configurations can share the same spec implementation:
```suggestion
  configurations (React to Rails migration for example).
```
See for example `packs/auth/spec/controllers/auth/sessions_controller_spec.rb`.